hal/generate_usage_reporting.py

```diff
@@ -3,7 +3,6 @@
 # Copyright (c) FIRST and other WPILib contributors.
 # Open Source Software; you can modify and/or share it under the terms of
 # the WPILib BSD license file in the root directory of this project.
-
```

Comment:
Unrelated?

Reply:
i believe it was something that came up when attempting to merge from main

Reply:
Seems to have been a weird interaction between the merge and the formatting. Just manually add the line back to clean up the PR diff.